src/cudadecoder/online-endpoint.h

```diff
@@ -0,0 +1,173 @@
+// cudadecoder/online-endpoint.h
+
+// Copyright 2013   Johns Hopkins University (author: Daniel Povey)
+// Copyright 2021   NVIDIA (author: Daniel Galvez)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef KALDI_ONLINE2_ONLINE_ENDPOINT_H_
+#define KALDI_ONLINE2_ONLINE_ENDPOINT_H_
+
+#include <string>
+#include <vector>
+#include <deque>
+
+#include "base/kaldi-types.h"
+#include "itf/options-itf.h"
+#include "util/parse-options.h"
+
+namespace kaldi {
+/// @addtogroup  onlinedecoding OnlineDecoding
```

Comment:
Please put this into a different group. If there is none yet, `cudadecoder` will do.